src/Components/Aspire.Confluent.Kafka/AspireKafkaConsumerExtensions.cs

```diff
@@ -28,8 +28,110 @@ public static class AspireKafkaConsumerExtensions
     /// <param name="configureSettings">An optional method for customizing the <see cref="KafkaConsumerSettings"/>.</param>
     /// <param name="configureBuilder">An optional method used for customizing the <see cref="ConsumerBuilder{TKey,TValue}"/>.</param>
     /// <remarks>Reads the configuration from "Aspire:Kafka:Consumer" section.</remarks>
-    public static void AddKafkaConsumer<TKey, TValue>(this IHostApplicationBuilder builder, string connectionName, Action<KafkaConsumerSettings>? configureSettings = null, Action<ConsumerBuilder<TKey, TValue>>? configureBuilder = null)
-        => AddKafkaConsumer(builder, DefaultConfigSectionName, configureSettings, configureBuilder, connectionName, serviceKey: null);
+    public static void AddKafkaConsumer<TKey, TValue>(this IHostApplicationBuilder builder, string connectionName, Action<KafkaConsumerSettings>? configureSettings, Action<ConsumerBuilder<TKey, TValue>>? configureBuilder)
```

Comment:
1. Can we put the overloads in some sort of logical order? Potentially from simplest to most complex? And group all the `AddKafkaConsumer` overloads together and all the `AddKeyedKafkaConsumer` overloads together.

2. Can we use `/// <inheritdoc/>` so we don't have to duplicate all the XML comments so many times?

Reply:
done!